plugins/source/aws/codegen/recipes/servicequota.go

```diff
@@ -0,0 +1,34 @@
+package recipes
+
+import (
+	"github.com/aws/aws-sdk-go-v2/service/servicequotas/types"
+	"github.com/cloudquery/plugin-sdk/codegen"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func ServiceQuotasResources() []*Resource {
+	resources := []*Resource{
+		{
+			SubService: "quotas",
```

Comment:
Maybe add a "services" (`aws_servicequotas_services`) table so that quota resolving can be parallelized?